core-api/src/test/resources/config/no-audience-project-config.json

```diff
@@ -1,13 +1,14 @@
 {
   "accountId": "789",
   "projectId": "1234",
-  "version": "4",
+  "version": "2",
```

Comment:
We still need to continue supporting version 1. We should have separate files and make sure everything works.